app/controllers/specialist_documents_controller.rb

```diff
@@ -5,9 +5,10 @@ class SpecialistDocumentsController < ApplicationController
 
   def show
     artefact = content_api.artefact(params[:path])
+    finder = Finder.new(content_store.content_item("/#{params[:path].split('/')[0]}"))
```

Comment:
What about leaving this in the `finder` method and just drop the `slug` argument, rather than inlining it?